trojsten/top30/tests.py

```diff
@@ -56,14 +56,23 @@ def setUp(self):
         self.group.user_set.add(self.users[3])
         self.users[2].groups.add(self.group)
         self.users[3].groups.add(self.group)
-        competition = Competition.objects.create(
-            name="TestCompetition", organizers_group=self.group
-        )
-        competition.sites.add(self.site)
+
         self.start_time_old = timezone.now() + timezone.timedelta(-10)
         self.start_time_new = timezone.now() + timezone.timedelta(5)
         self.end_time_old = timezone.now() + timezone.timedelta(-5)
         self.end_time_new = timezone.now() + timezone.timedelta(10)
+
+        self.site = Site.objects.get(pk=settings.SITE_ID)
+        self.task1, self.task2 = self._create_competition(self.site)
+        self.other_site = Site.objects.create(domain="ina.sutaz.sk", name="Another site")
+        self.other_task1, self.other_task2 = self._create_competition(self.other_site)
+        self.url = reverse("view_leaderboard")
+
+    def _create_competition(self, site):
+        competition = Competition.objects.create(
+            name="TestCompetition", organizers_group=self.group
+        )
+        competition.sites.add(site)
```

Comment:
```suggestion
        competition.sites.add(site)
        competition.save()
```

Netreba tu toto? V susi/kms testoch to vacsinou je ked pridavas takto vlastnosti objektom

Reply:
Neviem, v mnohých iných testoch tiež `save()` nie je a funguje to.

Reply:
Akože pokiaľ to funguje tak to nie je problém, iba mi to prišlo iné ako v kms a suši testoch tak som to tu spomenul.